sdk/parents/azure-sdk-parent/pom.xml

```diff
@@ -283,6 +283,22 @@
           <artifactId>maven-install-plugin</artifactId>
           <version>3.1.3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-install-plugin;external_dependency} -->
         </plugin>
+
+        <plugin>
```

Comment:
Do we want this to run in every pipeline? Right now, this will be configured that way as the check is always enabled. This could cause grief if one of our common dependencies suddenly has a CVE filed against it where this will cause _**EVERY**_ CI job to start failing until the dependency is update, which is something I wouldn't prefer.

I'd rather have this as part of the global aggregate-reports job only where we can watch the outputs and deal with CVEs accordingly, as there are many times where libraries like Netty or Jackson have a CVE on code we never use so we shouldn't eagerly force an upgrade on those.

Reply:
@alzimmermsft This plugin is currently **not** set to run in CI, as the details around the database download issue make it difficult to decide where to place it. It may potentially exist inside its own pipeline. Whatever the case, it won't be added in a manner that block any other pipeline.

Reply:
Yep, just realized this was defined in `pluginManagement` and not within `plugins`, so this won't run unless we explicitly request it to run or somewhere adds this plugin into `plugins`.